tests/test_err.py

```diff
@@ -31,22 +31,22 @@ def test_cplerror_str():
     assert str(err) == "test123"
 
 
-def test_issue2353(caplog, path_rgb_byte_tif):
+def test_issue2353(caplog):
     """Ensure transformer doesn't leave errors behind."""
     from rasterio.warp import calculate_default_transform
 
-    with rasterio.open(path_rgb_byte_tif) as src:
+    with rasterio.open("tests/data/goes.tif") as src:
         _ = src.colorinterp
         t, w, h = calculate_default_transform(
-            'PROJCS["unknown",GEOGCS["unknown",DATUM["unknown",SPHEROID["GRS 1980",6378137,298.257222096042]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433]],PROJECTION["Geostationary_Satellite"],PARAMETER["central_meridian",-137],PARAMETER["satellite_height",35786023],PARAMETER["false_easting",0],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",EAST],AXIS["Northing",NORTH],EXTENSION["PROJ4","+proj=geos +sweep=x +lon_0=-137 +h=35786023 +x_0=0 +y_0=0 +ellps=GRS80 +units=m +no_defs"]]',
+            src.crs,
             "EPSG:4326",
-            21696,
-            21696,
-            -5434894.885056,
-            -5434894.885056,
-            5434894.885056,
-            5434894.885056,
+            src.width,
+            src.height,
+            *src.bounds,
+        )
+        assert (
+            "Point outside of projection domain" in caplog.text or
+            "tolerance condition error" in caplog.text
         )
-        assert "Point outside of projection domain" in caplog.text
         assert "Encountered points outside of valid dst crs region" in caplog.text
```

Comment:
@sgillies locally the test fails here because it can find it in the logs 🤷‍♂️ I'm not sure why!